web/ui/react-app/src/Loader.tsx

```diff
@@ -0,0 +1,7 @@
+import React, { FC } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+
+const Loader: FC<{}> = () => <FontAwesomeIcon icon={faSpinner} spin />;
```

Comment:
@boyskila is refactoring these fetching / status indicator components in https://github.com/prometheus/prometheus/pull/6267. Probably it makes sense for that one to be merged first (feel free to leave input there!) and then adjust this here accordingly?

Reply:
I have some concerns about the Fetch component. I'll bring them up in the other PR review. I would prefer to keep `useFetch`

Reply:
Actually when this component was used, I thought it was loading something, but it's just a loading indicator. Could you rename it to `<Spinner>`, `<LoadingSpinner>`, `<LoadingIndicator>` or something like that to make usage code clearer?

Reply:
(though I still think we need to come to a conclusion about the abstractions introduced in that other PR, and whether they should be used here - I think at least after this wave of PRs we should focus on settling some things and making them consistent)

Reply:
I will remove the abstraction for the time being

Reply:
done